src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/ApprenticeController.cs

```diff
@@ -43,7 +43,7 @@ public async Task<IActionResult> Details(DetailsRequest request)
         }
 
         [HttpGet]
-        [Route("{apprenticeshipHashedId}/change-employer/inform", Name = RouteNames.ApprenticeInform)]
+        [Route("{apprenticeshipHashedId}/change-employer", Name = RouteNames.ApprenticeInform)]
```

Comment:
Tiny comment, if we are using RouteNames. Then it may be better to call this

 ApprenticeChangeEmployerInform